apps/web/src/services/dashboards/widgets/compliance-check-status/ComplianceCheckStatusWidget.vue

```diff
@@ -101,98 +99,155 @@ import { useWidgetLifecycle } from '@/services/dashboards/widgets/_hooks/use-wid
 
 /* Api */
-const fetchData = async (): Promise<Data[]> => {
+const fetchData = async (): Promise<Data> => {
     try {
         const apiQueryHelper = new ApiQueryHelper();
-        apiQueryHelper.setFilters(state.budgetConsoleFilters);
-        const { results } = await SpaceConnector.clientV2.costAnalysis.budgetUsage.analyze({
+        apiQueryHelper.setFilters(state.consoleFilters);
+        const { results } = await SpaceConnector.clientV2.inventory.cloudService.analyze({
             query: {
-                granularity: state.options.granularity,
-                start: state.dateRange.start,
-                end: state.dateRange.end,
                 fields: {
-                    total_spent: {
-                        key: 'usd_cost',
+                    compliance_count: {
+                        operator: 'count',
+                    },
+                    pass_check_count: {
+                        key: 'data.stats.checks.pass',
                         operator: 'sum',
                     },
-                    total_budget: {
-                        key: 'limit',
+                    fail_check_count: {
+                        key: 'data.stats.checks.fail',
                         operator: 'sum',
                     },
-                    budget_count: {
-                        operator: 'count',
+                    pass_score: {
+                        key: 'data.stats.score.pass',
+                        operator: 'sum',
+                    },
+                    total_score: {
+                        key: 'data.stats.score.total',
+                        operator: 'sum',
+                    },
+                    accounts: {
+                        key: 'account',
+                        operator: 'add_to_set',
+                    },
+                },
+                select: {
+                    compliance_count: 'compliance_count',
+                    pass_check_count: 'pass_check_count',
+                    fail_check_count: 'fail_check_count',
+                    account_count: {
+                        key: 'accounts',
+                        operator: 'size',
+                    },
+                    score: {
+                        operator: 'multiply',
+                        fields: [
+                            {
+                                operator: 'divide',
+                                fields: [
+                                    'pass_score',
+                                    'total_score',
+                                ],
+                            },
+                            100,
+                        ],
+                    },
+                },
+                ...apiQueryHelper.data,
+            },
+        });
+        if (results?.length) return results[0];
+        return {};
+    } catch (e) {
+        ErrorHandler.handleError(e);
+        return {};
+    }
+};
+const fetchSeverityData = async (): Promise<SeverityData[]> => {
```

Comment:
description: `severityData` is for nested donut chart!
Since this data is relatively less important, I separated it from the `state.data`.